oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxHWDiskStore.java

```diff
@@ -312,11 +312,11 @@ private static void computeDiskStats(LinuxHWDiskStore store, String devstat) {
     }
 
     private static String getPartitionNameForDmDevice(String vgName, String lvName) {
-        return new StringBuilder().append(DEV_LOCATION).append(vgName).append('/').append(lvName).toString();
+        return DEV_LOCATION + vgName + '/' + lvName;
```

Comment:
This compiles to the same thing.  I agree your version is more readable and don't mind the change as long as we're touching the file for other reasons...